tests/pytorch_pfn_extras_tests/onnx_tests/utils.py

```diff
@@ -51,6 +52,8 @@ def run_model_test(
                     output_names=output_names,
                     **kwargs,
                 )
+                torch_f.flush()
```

Comment:
Windows doens't allow multiple access to file so it needs to be closed instead
```suggestion
                torch_f.close()
```